RELEASE-NOTES.md

```diff
@@ -1,119 +1,135 @@
+* An info log has been added when starting a storage calculation batch. [[PR 1238](https://github.com/basho/riak_cs/pull/1238)]
+* `GET Bucket` requests now have clearer responses. A 501 stub for Bucket lifecycle and a  simple stub for Bucket requestPayment have been added. [[PR 1223](https://github.com/basho/riak_cs/pull/1223)]
+* Several user-friendly features have been added to [`riak-cs-debug`](http://docs.basho.com/riakcs/latest/cookbooks/command-line-tools/): fine-grained information gathering options, user-defined filtering for configuration files, and verbose output for failed commands. [[PR 1236](https://github.com/basho/riak_cs/pull/1236)]
+
+###Enterprise
+* MDC has `proxy_get`, which make block objects propagate to site clusters when they are requested. Now, multibag configuration with MDC supports `proxy_get`. [[PR 1171](https://github.com/basho/riak_cs/pull/1171) and [PR 25](https://github.com/basho/riak_cs_multibag/pull/25)]
+* Configuration options to replicate blocks in realtime replication (`replicate_cs_blocks_realtime`) and to replicate block tombstones in realtime replication (`replicate_cs_block_tombstone`) are now available. `replicate_cs_blocks_realtime` relieves the load in fullsync replication (there will be a longer realtime queue in realtime replication and risk of dropping manifests or other important keys), and it is set to `false` by default. `replicate_cs_block_tombstone` causes source block deletes caused by garbage collection to be replicated to the sink cluster. This option is set to `true` by default. Both configuration options are set in the advanced.config of the Riak node, rather than in the Riak CS configuration. [[PR 678](https://github.com/basho/riak_repl/pull/678)]
+* Several internal operation tools have been added to help diagnose or address
+  issues. [[PR 1145](https://github.com/basho/riak_cs/pull/1145),
+  [PR 1134](https://github.com/basho/riak_cs/pull/1134), and
+  [PR 1133](https://github.com/basho/riak_cs/pull/1133)]
+* Added a generic function for manual operations to resolve siblings of manifests and blocks, which will assist Basho Client Service Engineers with troubleshooting and solving issues. [[PR 1188](https://github.com/basho/riak_cs/pull/1188)]
+
+
+##Changes
+* Dependency versions have been updated in Riak S2 and Stanchion as follows: cuttlefish 2.0.4, node_package 2.0.3, riak-erlang-client 2.1.1, lager 2.1.1, lager_syslog 2.1.1, eper 0.92 (Basho patched), cluster_info 2.0.3, riak_repl_pb_api 2.1.1, and riak_cs_multibag 2.1.0. [[PR 1190](https://github.com/basho/riak_cs/pull/1190), [PR 1197 ](https://github.com/basho/riak_cs/pull/1197) , and [PR 27](https://github.com/basho/riak_cs_multibag/pull/27)].
```

Comment:
Lately we updated to lager 2.2.0 in https://github.com/basho/riak_cs/pull/1245 and https://github.com/basho/stanchion/pull/104 .